web/tests/input_pill.test.js

```diff
@@ -674,5 +674,5 @@ run_test("getCurrentText/onTextInputHook", ({mock_template}) => {
         preventDefault: noop,
     });
 
-    assert.deepEqual(widget.items(), [items.blue, items.red, items.yellow]);
+    assert.deepEqual(widget.items(), [items.blue, items.red]);
```

Comment:
...and is what necessitates this test change -- which means that we're no longer actually testing the typing ` ,` closes the pill at all.

Reply:
Yeah you are correct should have created a new test for new `input` event handler, I actually did this change for getting 100% line coverage.

Reply:
This gets _coverage_, but makes the tests misleading, removes an actually useful check, and does not actually check the new behaviour.
 
Revert this part of your change, and add a new test which actually tests the behaviour you changed.

Reply:
Yes yes doing it 👍

Reply:
Raised the PR adding new test: #29933